evap/evaluation/templates/base.html

```diff
@@ -159,27 +167,26 @@
             };
             applyTomSelect(document.querySelectorAll("select:not(.form-template select):not(.no-tomselect)"));
 
-            document.addEventListener('change', function(e){
-               $(e.target).removeClass("invalid")
-            }, true);
+            document.addEventListener('change', e => e.target.classList.remove("invalid"), true);
 
             // activate clickable hover tables
-            $(".table-seamless-links [data-url]").each(function() {
-                const row = $(this);
-                const link = "<a href='" + row.data("url") + "'></a>";
-                row.children().wrapInner(link);
+            document.querySelectorAll(".table-seamless-links [data-url]").forEach(row => {
+                for(const table_data of row.children) {
+                    table_data.innerHTML = "<a href='" + row.dataset["url"] + "'>" + table_data.innerHTML + "</a>"
```

Comment:
Tricky question. I'd say no, but we should have a test?

If its a value of `data-url`, that means that is was part of an HTML attribute before, so it already can't contain unescaped `"` characters, which would be the only problem here?

Reply:
It can definitely include `'` which would suffice here:
```html
<span data-url="'></a><strong>hello</strong>"></span>
<div>
</div>

<script>
    document.querySelector("div").innerHTML = "<a href='" + document.querySelector("span").dataset.url + "'>";
</script>
```
Maybe we should replace this code with something like the following (not tested):
```javascript
const wrapper = document.createElement("a");
wrapper.href = row.dataset.url;
wrapper.append(...table_data.children);
table_data.replaceChildren(wrapper);
```

Reply:
Yup, works, thanks :)